### ISSUE TITLE
CVE-2022-35252

### ISSUE THREAD

**Issue by achifal** [nginxinc/docker-nginx#699]
```
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| STATUS     | CVE SEVERITY           | PACKAGE NAME | PACKAGE VERSION | CVE DESCRIPTION                                               |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
| Unapproved | Unknown CVE-2022-35252 | curl         | 7.83.1-r2       | https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-35252 |
+------------+------------------------+--------------+-----------------+---------------------------------------------------------------+
```

**Comment by zops** [comment]
me to

**Comment by vitoyucepi** [comment]
I think this issue was fixed in `nginx:1.22-alpine` and `nginx:1.23-alpine`.
@achifal could you check and close the issue?

**Comment by thresheek** [comment]
Fixed in current Alpine images.
